substrate/frame/revive/fixtures/contracts/terminate_and_send_to_argument.rs

```diff
@@ -28,6 +28,6 @@ pub extern "C" fn deploy() {}
 #[no_mangle]
 #[polkavm_derive::polkavm_export]
 pub extern "C" fn call() {
-	let eve = [5u8; 20];
-	api::terminate(&eve);
+	input!(beneficiary: &[u8; 20],);
+	api::terminate(&beneficiary);
```

Comment:
This code made assumptions about how the address is derived. So it had to be changed.